include/zfs_namecheck.h

```diff
@@ -43,6 +43,8 @@ typedef enum {
 	NAME_ERR_RESERVED,		/* entire name is reserved */
 	NAME_ERR_DISKLIKE,		/* reserved disk name (c[0-9].*) */
 	NAME_ERR_TOOLONG,		/* name is too long */
+	NAME_ERR_SELF_REFERENCE,	/* self path reference in filesystem is not allowed */
```

Comment:
lines are limited to 80 characters with 8-character tabs. I think we can elide "is not allowed"

Reply:
will fix it thanks